frontend/src/concepts/pipelines/content/createRun/contentSections/PipelineSection.tsx

```diff
@@ -11,13 +11,19 @@ import ImportPipelineButton from '~/concepts/pipelines/content/import/ImportPipe
 import { PipelineKF } from '~/concepts/pipelines/kfTypes';
 
 type PipelineSectionProps = {
+  onLoaded: (loaded: boolean) => void;
   value: PipelineKF | null;
   onChange: (pipeline: PipelineKF) => void;
 };
 
-const PipelineSection: React.FC<PipelineSectionProps> = ({ value, onChange }) => {
-  const [pipelines] = usePipelines();
-
+const PipelineSection: React.FC<PipelineSectionProps> = ({ onLoaded, value, onChange }) => {
+  const [pipelines, loaded] = usePipelines();
+  React.useEffect(() => {
```

Comment:
Personally I think this is the most straightforward solution to the issue, but I find kind of disconnected the idea of the pipeline form value already filled before having all the pipeline values loaded, I don't think our current approach is wrong but I don't think is optimal either, since there's a disconnection between the form value and the visual component itself, I would love to know the opinion of other review on this one.